src-crg/README.md

```diff
@@ -0,0 +1,115 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/lord/img/master/logo-slate.png" alt="Slate: API Documentation Generator" width="226">
+  <br>
+  <a href="https://travis-ci.org/lord/slate"><img src="https://travis-ci.org/lord/slate.svg?branch=master" alt="Build Status"></a>
```

Comment:
Maybe don't want to have reference to TravisCI in our docs?

Reply:
This is the vendored README file that comes with Slate — we will want to replace this entire thing with the docs needed on how to build this component of our docs.

Reply:
Oh right I see